lld/COFF/InputFiles.h

```diff
@@ -89,16 +89,16 @@ class InputFile {
   // An archive file name if this file is created from an archive.
   StringRef parentName;
 
-  // Returns .drectve section contents if exist.
-  StringRef getDirectives() { return directives; }
+  // Returns .drectve section(s) content if exist.
+  std::vector<StringRef> getDrectves() { return drectves; }
 
   COFFLinkerContext &ctx;
 
 protected:
   InputFile(COFFLinkerContext &c, Kind k, MemoryBufferRef m, bool lazy = false)
       : mb(m), ctx(c), fileKind(k), lazy(lazy) {}
 
-  StringRef directives;
+  std::vector<StringRef> drectves;
```

Comment:
Any reason for not calling this just `directives` for a more fluid reading of the code?